twitter/twitter.php

```diff
@@ -1423,7 +1429,7 @@ function twitter_media_entities($post, array &$postarray)
 	return '';
 }
 
-function twitter_createpost(App $a, $uid, $post, array $self, $create_user, $only_existing_contact, $noquote)
+function twitter_createpost(App $a, $uid, $post, array $self, $create_user, $only_existing_contact, $noquote, int $uriid = 0)
```

Comment:
What is the meaning ot `$uriid = -1`? Please add a function comment block to explicit this use.

Reply:
For the tags it is needed to know the correct `URI-ID`. But this function is called recursively and also for mirror posts (where we handle the tags differently). So an empty value means that the function creates the `URI-ID`. When the value isn't empty, that one is used for the tags. And when below `0` the tags aren't stored, since the function is called for the mirrored posts.